pyintegration/deephaven2/table.py

```diff
@@ -2,28 +2,85 @@
 #   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
 #
 """ This module implements the Table class and functions that work with Tables. """
+from __future__ import annotations
+
 from typing import List
 
+import jpy
+
 from deephaven2 import DHError
 from deephaven2.column import Column
+from deephaven2.combo_agg import ComboAggregation
+from deephaven2.constants import SortDirection
+
+TableTools = jpy.get_type("io.deephaven.db.tables.utils.TableTools")
+SortPair = jpy.get_type("io.deephaven.db.tables.SortPair")
```

Comment:
I have a general preference to use the newer java qst / table APIs where possible - ala, io.deephaven.api.SortColumn here. @rcaudy ?

Reply:
Tried to use SortColumn instead but ran into a potential JPY overload matching issue, pretty sure I don't want to touch JPY just yet.